index.html

```diff
@@ -18,12 +18,17 @@
         <header class="head">Uthman</header>
         <i class="fa-solid fa-bars menu-bar"></i>
```

Comment:
- [ ] Great job on adding the links for the desktop version, however the mobile menu should be hidden and only displayed in mobile view 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/206857241-59beb6dd-1391-4a84-9a81-849a826da65f.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/206857246-fbff8fa8-74a3-4089-9ace-27798066bdc1.png)